de.dlr.sc.virsat.swtbot.test/META-INF/MANIFEST.MF

```diff
@@ -11,6 +11,7 @@ Require-Bundle: org.eclipse.swt,
  org.eclipse.swtbot.go,
  org.eclipse.swtbot.junit4_x,
  org.eclipse.swtbot.swt.finder,
+ org.eclipse.swtbot.eclipse.gef.finder;bundle-version="2.8.0",

```

Comment:
At the moment we do not bind versions in Virtual Satellite when possible. please remove the specific version